ssz-rs/src/lib.rs

```diff
@@ -64,14 +67,13 @@ where
     T::deserialize(encoding)
 }
 
-#[derive(Debug, Error)]
-#[error("{0}")]
+#[derive(Debug)]
 pub enum SimpleSerializeError {
-    Serialize(#[from] SerializeError),
-    Deserialize(#[from] DeserializeError),
-    Merkleization(#[from] MerkleizationError),
-    List(#[from] ListError),
-    Vector(#[from] VectorError),
+    Serialize,
+    Deserialize,
+    Merkleization,
+    List,
+    Vector,
 }
```

Comment:
should be straightforward to add a `fmt::Display` impl to retain backwards compatibility with `thiserror` functionality

Reply:
Totally, good point. Implemented in https://github.com/ralexstokes/ssz-rs/pull/25/commits/ea113b48f3ad134603f11c9be9f5fc696d30c259.